framework/assets/yii.gridView.js

```diff
@@ -123,7 +123,7 @@
             var $form = $('<form/>', {
                 action: url,
                 method: 'get',
-                class: 'gridview-filter-form',
+                'class': 'gridview-filter-form',
```

Comment:
Quoting single key, while leaving other as they are looks weird, unpretty.
Besides, do we realy need to wait until more of those keys become reserved?